libcloudforensics/providers/aws/internal/ec2.py

```diff
@@ -378,15 +380,6 @@ def GetOrCreateAnalysisVm(
       return instances[0], created
 
     instance_type = common.GetInstanceTypeByCPU(cpu_cores)
-    startup_script = utils.ReadStartupScript()
-    if packages:
-      startup_script = startup_script.replace('${packages[@]}', ' '.join(
-          packages))
-
-    # Install ec2-instance-connect to allow SSH connections from the browser.
-    startup_script = startup_script.replace(
```

Comment:
Mmmh what happens here if `userdata` below is not provided? IIRC if you don't install `ec2-instance-connect` then you won't be able to SSH to the machine from the browser. Has this behaviour change? It looks like now the default behaviour is to not provide a startup script anymore. Is this assumption correct?

Reply:
See other pr convo.